deploy/ansible/roles/docker/tasks/ubuntu.yml

```diff
@@ -32,4 +30,28 @@
       name: docker-ce={{ dockerVersion }}
       state: present
       update_cache: yes
-  
\ No newline at end of file
+
+# purge if anything installed to resolve conflicts
+  - name: remove existing python modules
+    pip:
+      name: "{{ item }}"
+      state: absent
+    with_items:
+      - docker
+      - docker-py
+      - docker-compose
+
+  - name: install docker python modules
+    pip:
+      name: "{{ item }}"
+      state: latest
+    with_items:
+      - docker
+
+  - name: download docker-compose bin
+    raw: curl -L https://github.com/docker/compose/releases/download/{{ dockerComposeVersion }}/docker-compose-`uname -s`-`uname -m` -o {{ execPath }}/docker-compose
```

Comment:
Why not installing from PyPi with PIP?

Reply:
@bakhtin multiple conflicts that I did not resolve in adequate time